Programming-in-Python.md

```diff
@@ -93,6 +93,7 @@
 - [SO: Python coding standards/best practices](https://stackoverflow.com/questions/356161/python-coding-standards-best-practices)
 - [Python Best Practices: 5 Tips For Better Code - Airbrake Blog](https://airbrake.io/blog/python/python-best-practices)
 - [Python tutorial: Best practices and common mistakes to avoid](https://jaxenter.com/python-tutorial-best-practices-145959.html)
+- [Common mistakes beginnners make in python](https://github.com/qxf2/wtfiswronghere)
```

Comment:
You can also add this link to the Beginners / Learning section of this page - as a separate PR, I think it also applies there (also check if it can be added elsewhere as well)

Reply:
Wouldn't it be a duplicate adding across sections?

Reply:
Good question and my response:

- if the links are spread across to relevant categories and sections where they do apply, then it's not duplication at all, if it's in the same section or category then they are duplicate.
- by adding to multiple categories/sections we highlight the links have multiple facets and are important to look at under those contexts - especially it's an important resource and it brings value to those who rely on our sharings
- we don't want anyone to miss anything, as far as we can - there are so many places I still want to cross-link but I'm not finding the opportunity for it

I hope it answers your question.